lib/reverb/Reverb.cc

```diff
@@ -454,7 +454,8 @@ void MixxxPlateX2::processBuffer(const sample_t* in, sample_t* out, const uint f
 		sample_t mono_sample = blend * (in[i] + in[i + 1]) / 2;
 		sample_t xl, xr;
 		PlateStub::process(mono_sample, decay, &xl, &xr);
-		out[i] = xl + in[i];
-		out[i + 1] = xr + in[i + 1];
+		out[i] = (xl * wet) + (in[i] * (1 - wet));
```

Comment:
We loose the original reverb sound now. if we set wet to 0.5, we have the original sound but with -6 dB.